packages/clerk-js/src/core/clerk.ts

```diff
@@ -500,6 +504,10 @@ export default class Clerk implements ClerkInterface {
       }
     }
 
+    if (su.status == 'missing_requirements') {
```

Comment:
Is this scenario possible? Let's take a potential user "A" for example:

- If A does not exist and we start from si, this is going to be handled by line 478
- If A does not exist and we start from su, this is going to be handled by the component (no transfer will take place)
- If  A exists and we start from si, we cannot end up in a  su `missing_requirements` state*
- If A` exists and we start from su, we cannot end up in a  su `missing_requirements` state*

(*) This could be possible if the customer updates the instance settings and adds more su requirements after A signed up. In this case, if A tries to sign in, would we prompt them to enter the additional required fields would we simply complete the sign in and let the customer handle the extra data?

Reply:
This scenario occurs when the oauth flow completes (for a sign-up) & there are missing requirements (e.g. the fname & lname are mandatory but the provider doesn't return them, or the provider does not return a verified email).